bar/90_contents/UpdateEngine/updateVEvent.js

```diff
@@ -583,3 +587,26 @@ function getAccessToken(bodyData) {
   var res = JSON.parse(response.body);
   return res.access_token;
 }
+
+function checkParams(request, params){
+  try {
```

Comment:
Exception does not occur in this method, you don't need to do try/catch I think.

Reply:
I understand.